main.go

```diff
@@ -82,6 +83,7 @@ func parseFlags() (*config, error) {
 	flag.StringVar(&cfg.oidc.audience, "oidc.audience", "", "The audience for whom the access token is intended, see https://openid.net/specs/openid-connect-core-1_0.html#IDToken.")
 	flag.StringVar(&cfg.oidc.username, "oidc.username", "", "The username to use for OIDC authentication. If both username and password are set then grant_type is set to password.")
 	flag.StringVar(&cfg.oidc.password, "oidc.password", "", "The password to use for OIDC authentication. If both username and password are set then grant_type is set to password.")
+	flag.StringVar(&cfg.scope, "scope", "", "The scope to be included in the payload data of the token.")
```

Comment:
Probably a good idea to use `flag.StringSliceVar` here, as there might be multiple scopes to include?

Reply:
Multiple scopes are supported by a single "scope" key, and the value being a string with each scope separated by spaces.  If you simply change to StringSliceVar and the corresponding data types to []string, it tries to send multiple `scope` parameters, you get the following error:

```
level=error name=token-refresher ts=2023-09-18T15:08:05.355519306Z caller=main.go:244 msg="failed to get token" err="oauth2: cannot fetch token: 400 Bad Request\nResponse: {\"error\":\"invalid_request\",\"error_description\":\"duplicated parameter\"}"
```

My latest change now supports multiple scopes in the following ways:

`--scope="openid offline_access"`
`--scope="openid,offline_access"`
`--scope="openid" --scope="offline_access"`